src/otx/v2/adapters/torch/mmengine/engine.py

```diff
@@ -32,6 +32,39 @@
 
 logger = get_logger()
 MMENGINE_DTYPE = ("float16", "bfloat16", "float32", "float64")
+# NOTE: We need to discuss where to declare the default value.
+DEFAULT_CONFIG: dict[str, dict | list | str | int | float | None] = {
```

Comment:
Just a question. Can't we just use `dict` or `dict[str, Any]`?
Looks too verbose.

Reply:
Any is now disabled by ruff, and I've also annotated this because mypy wants it.

Reply:
We should probably use Config directly instead of this, I'll fix it.

Reply:
I modified this from a dict to a Config.